packages/store/src/patterns/encodeKey.js

```diff
@@ -140,6 +144,94 @@ const dbEntryKeyToBigint = k => {
   }
 };
 
+// `'\u0000'` is the terminator after elements.
+// `'\u0001'` is the backslash-like escape character, for
+// escaping both of these characters.
```

Comment:
Do we have a suitable reservoir of Unicode pedantry in the house to be sure this encoding doesn't alias to something else weird if these codes precede or follow any other value?

Reply:
Yeah, I'm confident enough.